openapi/wicketd.json

```diff
@@ -10,6 +10,60 @@
     "version": "0.0.1"
   },
   "paths": {
+    "/artifacts/{name}/{version}/{hash}": {
```

Comment:
I don't think we'd want to require a hash for all artifacts. It's really only needed for phase2 images. I'm not sure how those will be versioned. Ideally, we would just target by name and version. But in some cases we may want name and hash. For now, I'd just exclude the hash.

Reply:
Done